docs/tutorials/migrations/homebrew.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Homebrew
+description: Common questions and solutions for using Rust with Flox
+---
+
+# Homebrew migration guide
+Flox is a virtual environment manager and package manager all in one. Using Flox, you can create virtual environments containing packages from the Flox Catalog.
+
+Flox can replace Homebrew entirely, or they can be used together.
+
+This guide explains how to introduce Flox into environments where Homebrew is currently being used, either as a replacement or an addition. It introduces new concepts and proposes a basic procedure for mapping packages.
+
+## Why you might want to migrate
+Homebrew does a great job, and has been loved as the “missing package manager” for a generation of macOS users. But there are a few reasons you might consider moving to Flox:
+
+* **You need a cross-platform package manager.** Flox works on both macOS and Linux - x86 and ARM - allowing you to define a set of packages that works the same everywhere.
+* **You need a virtual package manager** Flox allows you to create as many environments as you need, each with a different set of packages.
+* **You need reproducible environments.** Flox environments are defined in a TOML manifest that can be easily shared, or managed alongside code.
+* **You need a broader set of software.** Homebrew has a sizeable collection of packages, but the Flox Catalog is based on Nixpkgs, the largest collection of packages in the world.
+* **You need older versions of software.** The Flox Catalog contains historical versions of each of its packages - in many cases, going back decades.
+
+## Install Flox
+Download and install Flox following the [installation instructions]().
+
+## Find and install your first package
+Migrating to Flox is a straightforward process of installing Flox packages for each of your Homebrew formulae. This section walks you through the process for identifying the set of Homebrew formulae you currently have installed, searching for the corresponding packages in the Flox Catalog, and installing them.
+
+As a Homebrew user, you will find several of the Flox subcommands familiar:
+
+* `search` is used to find available packages
+* `install` is used to install packages
+* `uninstall` is used to remove packages
+
+### Show top-level formulae in Homebrew
```

Comment:
We don't mention how to migrate if you have a `Brewfile`.  That might be a really small note, but I haven't played with them too much.

Reply:
Good point. I think we should do a bit more investigation and add this as a second pass.